app/assets/javascripts/components/Menu/Menu.tsx

```diff
@@ -117,7 +118,7 @@ export const Menu: FunctionComponent<MenuProps> = ({
       style={style}
       aria-label={a11yLabel}
     >
-      {Array.isArray(children) ? children.map(mapMenuItems) : null}
+      {toChildArray(children).map(mapMenuItems)}
```

Comment:
Will this work as prevously if there's only one child node? According to their (not very clear) [docs](https://preactjs.com/guide/v10/api-reference/#tochildarray) `toChildArray` will return an array with single element, but with our previous logic we return `null`.